### PR TITLE
DOC: Tim orcid id

### PR DESCRIPTION
**Pull request by chiahaoliu**

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -419,34 +419,26 @@ sbillinge:
...
-    - category: Markup Languages
-      level: intermediate
-      name: LaTeX
-    - category: Specialized Software
-      level: intermediate
-      name: Mathematica
-    - category: Specialized Software
-      level: intermediate
-      name: MATLAB
-  title: Mr.
```
Comment (sbillinge) [inline db/people.yml]: I think you should leave this line....deleted by mistake?

File: db/people.yml [hunk 2]
```diff
@@ -419,34 +419,26 @@ sbillinge:
...
     - Liu, C-H.
-  avatar: https://avatars0.githubusercontent.com/u/15218640?s=400&v=4
   education:
     - begin_year: 2014
       degree: MS/PhD in Applied Physics
       institution: Columbia University
       department: apam
       other:
         - 'Adviser: Simon Billinge'
+      group: bg
```
Comment (sbillinge) [inline db/people.yml]: Thanks for adding this!

File: db/people.yml (hunk 1 above)
Comment (chiahaoliu) [reply db/people.yml]: Yeah I add it back. Thanks for reminding!

File: db/people.yml [hunk 3]
```diff
@@ -423,6 +423,7 @@ chliu:
     - Chia-Hao Liu
     - Liu, Chia-Hao
     - Liu, C-H.
+  title: ''
```
Comment (sbillinge) [inline db/people.yml]: this should be 'Mr', or whatever pronoun you prefer

**Comment by sbillinge** [comment]
thanks @chiahaoliu

**Pull request merged**